index.html

```diff
@@ -13,8 +13,8 @@
 		<div class="wrapper">
 			<!-- This box contains the large image and text first seen on page-->
 			<div class="box1">
-				<p>This will be an image</p>
 				<h1>Skinclusive<br>inclusive of your skin.</h1>
+				<a href="#">Take the quiz now  ></a>

```

Comment:
This version includes a button underneath the large centred text that takes the user to the skin type quiz.